tests/test_core.py

```diff
@@ -7256,7 +7260,8 @@ def setUp(self):
 binaryen3jo = make_run("binaryen3jo", compiler=CLANG, emcc_args=['-O3', '-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="interpret-binary,asmjs"'])
 
 # This only works when .emscripten specifies a JS_ENGINE with native wasm support.
-binaryen_native = make_run("binaryen_native", compiler=CLANG, emcc_args=['-O2', '-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"'])
+binaryen0_native = make_run("binaryen0_native", compiler=CLANG, emcc_args=['-O0', '-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"'])
```

Comment:
Sounds good. Does the binaryen test suite cover the integration between the asm.js-compiled interpreter and the emscripten JS glue?  I did go ahead and leave a `binaryen2_interpret` mode that we can use on demand to test that if we want.